ibm/service/power/data_source_ibm_pi_volume_groups.go

```diff
@@ -0,0 +1,131 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package power
+
+import (
+	//"fmt"
+
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+)
+
+func DataSourceIBMPIVolumeGroups() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPIVolumeGroupsRead,
+		Schema: map[string]*schema.Schema{
+			helpers.PICloudInstanceId: {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			// Computed Attributes
+			"volume_groups": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"volume_group_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"replication_status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"consistency_group_name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"status_description": {
+							Type:     schema.TypeList,
```

Comment:
Not a list?

Reply:
refactored the status description attribute status_description_erros